Autopilot/AttitudeManager/FixedWing.c

```diff
@@ -0,0 +1,248 @@
+/*
+ * File:   FixedWing.c
+ * Author: Chris Hajduk
+ *
+ * Created on July 2, 2015, 8:04 PM
+ */
+
+#include "PWM.h"
+#include "AttitudeManager.h"
+#include "delay.h"
+#include "FixedWing.h"
+#include "ProgramStatus.h"
+
+#if VEHICLE_TYPE == FIXED_WING
```

Comment:
This is required. The header files are only included if their flag is set, but the source files are all compiled and linked regardless, as they are still in the project.